services/worker/src/worker/utils.py

```diff
@@ -33,6 +34,9 @@
 from libcommon.exceptions import NormalRowsError, StreamingRowsError
 from libcommon.utils import orjson_dumps
 
+MAX_IMAGE_PIXELS = 1_000_000_000
```

Comment:
Do you think it is good to have this constant here instead of in a config file?

Reply:
I was wondering if we really want to configure this, or if it's OK to hardcode. For now, I would say it's OK. Maybe we could move it to a constants.py file if it's cleaner (I think it's not the only hardcoded value not in a constants.py file).

Reply:
ok to have it here imo, but if it's used somewhere else in the future we can move it